make_queue/static/make_queue/css/quota_panel.css

```diff
@@ -0,0 +1,5 @@
+.noquota {
```

Comment:
No need to add custom CSS for this. Can use the `<i>...</i>` tag for italic and there is a global CSS class named `text-centered` for centering text.